rts/sra/src/main/java/io/narayana/sra/demo/api/HotelController.java

```diff
@@ -56,7 +56,7 @@ public class HotelController extends SRAParticipant {
     @Path("/book")
     @Produces(MediaType.APPLICATION_JSON)
     @SRA(SRA.Type.REQUIRED)
-    public Booking bookRoom(@HeaderParam(RTS_HTTP_CONTEXT_HEADER) String sraId,
+    public Booking bookRoom(@HeaderParam(RTS_HTTP_CONTEXT_HEADER) @DefaultValue("") String sraId,
```

Comment:
We cant set a default value for sraId as per Tom, because a transaction can't have a "default" ID. It needs to be allocated by the transaction manager.